app.js

```diff
@@ -1,111 +0,0 @@
-//Event listener on remove button for each element
-var removeElements = document.getElementsByClassName('container__elements__button');
```

Comment:
There is a chance the that DOM is not ready by the time all this script is run.
Please read about async & defer scripts here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script